synapse/models/inet.py

```diff
@@ -252,6 +252,70 @@ def _normPyStr(self, valu):
 
         return valu, {'subs': subs}
 
+class Srv4(s_types.Type):
```

Comment:
I think these types should go away because of inet:addr ( and inet:client / inet:server ).  Thoughts?

Reply:
concur